app/policies/equipment_policy.rb

```diff
@@ -2,19 +2,19 @@ class EquipmentPolicy < ManagedModelsPolicy
   def new?
     return false unless user
```

Comment:
I know that this was already in the code, but in the future (not for this PR) we should eliminate this duplications with a before_action callback